File: app/src/main/java/com/callmangement/ui/pos_issue/adapter/ViewIssuesActivityAdapter.java
package com.callmangement.ui.pos_issue.adapter;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.ViewGroup;
import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.callmangement.databinding.ItemViewIssuesBinding;

public class ViewIssuesActivityAdapter extends RecyclerView.Adapter<ViewIssuesActivityAdapter.ViewHolder> {
    private final Context context;

    public ViewIssuesActivityAdapter(Context context) {
        this.context = context;
    }

    @NonNull
    @Override
    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        ItemViewIssuesBinding binding = ItemViewIssuesBinding.inflate(LayoutInflater.from(parent.getContext()),parent,false);
        return new ViewHolder(binding);
    }

    @Override
    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {

    }

    @Override
    public int getItemCount() {
        return 5;
    }

    public static class ViewHolder extends RecyclerView.ViewHolder{
        private final ItemViewIssuesBinding binding;
        public ViewHolder(ItemViewIssuesBinding binding) {
            super(binding.getRoot());
            this.binding = binding;
        }
    }
}

File: app/src/main/java/com/callmangement/ui/pos_issue/adapter/ViewIssuesActivityAdapter.kt
package com.callmangement.ui.pos_issue.adapter

import android.content.Context
import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.callmangement.databinding.ItemViewIssuesBinding

class ViewIssuesActivityAdapter(private val context: Context) :
    RecyclerView.Adapter<ViewIssuesActivityAdapter.ViewHolder>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
        val binding =
            ItemViewIssuesBinding.inflate(LayoutInflater.from(parent.context), parent, false)
        return ViewHolder(binding)
    }

    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
    }

    override fun getItemCount(): Int {
        return 5
    }

    class ViewHolder(private val binding: ItemViewIssuesBinding) : RecyclerView.ViewHolder(
        binding.root
    )
}
